libs/langchain/langchain/hub.py

```diff
@@ -33,7 +36,7 @@ def push(
     api_url: Optional[str] = None,
     api_key: Optional[str] = None,
     parent_commit_hash: Optional[str] = "latest",
-    new_repo_is_public: bool = True,
+    new_repo_is_public: bool = False,
```

Comment:
possible to make these `Optional[bool] = None` and have the defaults encoded in sdk?

Reply:
yup